src/app/create-event/page.tsx

```diff
@@ -27,12 +27,7 @@ type DefaultValues = {
 };
 
 const CreateEventPage: NextPage = () => {
-  const {
-    register,
-    watch,
-    handleSubmit,
-    formState: { errors },
-  } = useForm({
+  const methods = useForm<DefaultValues>({
```

Comment:
Please don't call them methods, they are functions :D. Methods are in objects, we don't use that abomination here. Also, again and again. 

Refrain from your favorite pattern - calling stuff extremely generic way. I don't mean we need `createEventFunctions` here, that would be silly, but `formProps` would be a nice addition instead of `functions`/`methods`/`stuff`/`x` or anything like this.

You can take inspiration in their docs, as they call it.

![CleanShot 2023-11-16 at 11 19 06](https://github.com/GLObus303/sportujspolu/assets/12488648/1df397f9-b2c4-4cdd-8f96-e04d5f47ef4a)

formProps! :D (I actually didn't know)